CrosswordMe.py

```diff
@@ -1,95 +1,208 @@
-from random import randint
+#Fill in the words in line 32 you want to be set
+#Comment out fill_field_with_randoms call in the end to see where your words are being set (will be at a different postion the next time, since a new puzzle will be generated)
+
 from random import sample
+from random import choice
+from string import ascii_letters
+
 
 def print_puzzle(puzzle):
     #Prints the puzzle puzzle on a board
-    for line in range(line_count):
-        for position in range(column_count):
-            print(str(puzzle[line][position]) + " ", end="") #Prints character in current position
+    for puzzle_y_position in range(LINE_COUNT):
+        for puzzle_x_position in range(COLUMN_COUNT):
+            print(str(puzzle[puzzle_y_position][puzzle_x_position]).upper() + " ", end="") #Prints character in current position
         print() #Prints Newline
 
-def word_input():
-    #Collects the words for the puzzle
+def fill_field_with_randoms(puzzle):
+    #Fills the rest of the free space (represented by " ") with random letters
+
+    for puzzle_y_position in range(LINE_COUNT):
+        for puzzle_x_position in range(COLUMN_COUNT):
+
+            if puzzle[puzzle_y_position][puzzle_x_position] == " ": #If the position is empty
+                puzzle[puzzle_y_position][puzzle_x_position] = choice(ascii_letters)
+
 
-    words = ["one", "two", "three", "looooong"]
-    user_input = ""
+
+    return puzzle
+
+def word_input():
+    #Collects the words for the puzzle.
+    #Sorts the words by length descedning. Therefore the longes word is set first
+
+    words = ["Test", "Noch", "Nocheiner", "loooooooooooooooooooooong"]
+    for word in words:
+        if len(word) > LINE_COUNT:
+            print(word + " has " + str(len(word)) + " letters. A maximum length of " + str(LINE_COUNT) + " Letters is possible. " + word + " has been ommitted" )
+            words.remove(word)
```

Comment:
it is a bad idea to delete items from a list while iterating over the same list. Might not work if the longest word is not in the last position. You might want to do `for word in reversed(words)`. reversed() returns a new iterator in this case and `word` will be removed from the list `words`, but not from the iterator you are looping on